cypress/elements/DimensionModal/dataDimension.js

```diff
@@ -0,0 +1,81 @@
+import { DIMENSION_ID_DATA } from '@dhis2/analytics'
+import { expectDimensionModalToBeVisible } from '.'
+
+const unselectedListEl = 'data-dimension-item-selector-unselected-items-list'
+//const unselectedItemEl = 'data-dimension-item-selector-unselected-items-list-item'
+const dataTypesSelectButtonEl = 'data-dimension-data-types-select-field-content'
+//const selctedListEl = 'data-dimension-item-selector-selected-items-list'
```

Comment:
Suggest deleting unused code. Maybe you can store it in a separate branch for use later?

Reply:
Yeah, good point. I've removed most of the commented code and only kept some that are WIP, but I added a description to those for future reference.